.github/workflows/validate_and_publish_release.yml

```
@@ -42,6 +42,10 @@ jobs:
         id: install_python_packages
         run: |
           python -m pip install -f file://$PWD/artifact/ iree-compiler iree-runtime iree-tools-tflite iree-tools-tf
+      - name: Validate IREE Runtime Package
+        id: validate_runtime_package:
```

I think yamllint wants the trailing `:` removed here

```suggestion
        id: validate_runtime_package
```